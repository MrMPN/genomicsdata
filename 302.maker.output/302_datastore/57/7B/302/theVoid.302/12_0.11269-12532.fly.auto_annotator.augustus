# This output was generated with AUGUSTUS (version 2.5.5).
# AUGUSTUS is a gene prediction tool for eukaryotes written by Mario Stanke (mario.stanke@uni-greifswald.de)
# and Oliver Keller (keller@cs.uni-goettingen.de).
# Please cite: Mario Stanke, Mark Diekhans, Robert Baertsch, David Haussler (2008),
# Using native and syntenically mapped cDNA alignments to improve de novo gene finding
# Bioinformatics 24: 637-644, doi 10.1093/bioinformatics/btn013
# reading in the file /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/12_0.11269-12532.fly.auto_annotator.xdef.augustus ...
# Setting 1group1gene for E.
# Sources of extrinsic information: M P E 
# Have extrinsic information about 1 sequences (in the specified range). 
# Initialising the parameters ...
# fly version. Use default transition matrix.
# Looks like /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/12_0.11269-12532.fly.auto_annotator.augustus.fasta is in fasta format.
# We have hints for 1 sequence and for 1 of the sequences in the input set.
#
# ----- prediction on sequence number 1 (length = 1263, name = 302) -----
#
# Constraints/Hints:
302	EST-EXON	exonpart	202	872	0	-	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-EXON	exonpart	952	1063	0	-	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-INTRON	intronpart	874	950	0	-	.	grp=n_pieces;src=E "1;1;0:0"
302	PROTEIN	CDSpart	202	872	0	-	.	grp=p_pieces;src=P "100000;1;0:0"
302	PROTEIN	CDSpart	952	1063	0	-	.	grp=p_pieces;src=P "100000;1;0:0"
302	EST-EXON	genicpart	873	951	0	.	.	grp=n_peices;src=E "2;1;0:0"
# Predicted genes for sequence number 1 on reverse strand
# start gene g1
302	AUGUSTUS	gene	1	1063	0.31	-	.	g1
302	AUGUSTUS	transcript	1	1063	0.31	-	.	g1.t1
302	AUGUSTUS	intron	1	570	0.4	-	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	intron	863	951	0.71	-	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	571	862	0.51	-	2	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	952	1063	1	-	0	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	start_codon	1061	1063	.	-	0	transcript_id "g1.t1"; gene_id "g1";
# protein sequence = [MFIVAVTGGIATGKSTVTKVFERNGIPVIDADRIAREIVEPGQSCWQKIRATFGDEVLLPSKELNRAVLGRMIFENKE
# LRGKLNQITHPVIHRTIFWRVFKLFMSGHAWIVLDLPLLFETGILMDFIHKIVTVT]
# Evidence for and against this transcript:
# % of transcript supported by hints (any source): 50
# CDS exons: 1/2
#      E:   1 
#      P:   1 
# CDS introns: 1/2
#      E:   1 
# 5'UTR exons and introns: 0/0
# 3'UTR exons and introns: 0/0
# hint groups fully obeyed: 1
#      E:   1 (n_pieces)
# incompatible hint groups: 2
#      E:   1 (n_peices)
#      P:   1 (p_pieces)
# end gene g1
###
# command line:
# /home/genomica/Programs/maker/exe/augustus/bin/augustus --species=fly --strand=backward --UTR=off --hintsfile=/home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/12_0.11269-12532.fly.auto_annotator.xdef.augustus --extrinsicCfgFile=/home/genomica/Programs/maker/exe/augustus/config/extrinsic/extrinsic.MPE.cfg /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/12_0.11269-12532.fly.auto_annotator.augustus.fasta
