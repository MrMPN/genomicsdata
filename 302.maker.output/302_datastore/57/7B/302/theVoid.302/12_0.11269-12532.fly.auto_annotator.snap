>302 12_0 offset:11269
Einit	952	1063	-	21.952	0	1	2	302-snap.1
Exon	567	862	-	44.456	2	0	1	302-snap.1
Eterm	195	530	-	56.054	0	0	1	302-snap.1
